.github/workflows/style.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Code Style'
+
+on: pull_request
+
+jobs:
+  SwiftFormat:
+    runs-on: macOS-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: make lint
```

Comment:
I'd consider using a tool like rtx to ensure we are deterministically using the same Swiftformat version across environments. That's the setup we are following in the [tuist repository](https://github.com/tuist/tuist/blob/main/.rtx.toml)

Reply:
I'll look into it, but for now, I'm able to specify a `--minversion` configuration in `.swiftformat`.